habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -257,6 +257,8 @@ class HrlDefinedSkillConfig(HabitatBaselinesBaseConfig):
     turn_power_y: float = 0.0
     # The flag for using pick skill as the place skill
     use_pick_skill_as_place_skill: bool = False
+    # The flag for switching between point nav and social nav
+    hybrid_approach: bool = False
```

Comment:
Is this flag actually just for FetchBaselinesController and the HITL codebase? If so, we should avoid adding it to habitat-baselines. It could be a HITL-tool command-line arg. (In the future, we need to integrate Hydra configs properly for the HITL tool, itself, but that is outside the scope of this PR.)

Reply:
Very nice, thank you, Eric. I changed this to command-line arg.